src/test/java/com/redhat/cloud/policies/app/RestApiTest.java

```diff
@@ -588,21 +593,28 @@ void testGetPoliciesForUnknownAccount() {
 
     @Test
     void testGetOnePolicy() {
-        when(policiesHistoryRepository.getLastTriggerTime("1234", UUID.fromString("bd0ee2ec-eec0-44a6-8bb1-29c4179fc21c")))
-                .thenReturn(new GregorianCalendar(2020, 4, 10, 10, 0, 0).getTimeInMillis());
+        PoliciesHistoryEntry entry = new PoliciesHistoryEntry();
+        entry.setId(UUID.randomUUID());
+        entry.setTenantId(accountId);
+        entry.setPolicyId("9b3b4429-1393-4120-95da-54c17a512367");
+        entry.setCtime(new GregorianCalendar(2020, 4, 10, 10, 0, 0).getTimeInMillis());
+        Transaction transaction = session.beginTransaction();
+        session.persist(entry);
+        session.flush();
+        transaction.commit();
 
         JsonPath jsonPath =
                 given()
                         .header(authHeader)
-                        .when().get(API_BASE_V1_0 + "/policies/bd0ee2ec-eec0-44a6-8bb1-29c4179fc21c")
+                        .when().get(API_BASE_V1_0 + "/policies/9b3b4429-1393-4120-95da-54c17a512367")
```

Comment:
Is there a reason why you had to switch to that policy instead of using the old one?

Reply:
It seems that the tests are entagled here - so updating that policy also updates the `mtime` and other test fails because it expects it to be first when sorting my `mtime`.